server/README.md: change `docker-compose` description (#335)
eclipse-basyx/basyx-python-sdk: server/README.md

## Building and running the image with docker-compose

The container image can also be built and run via:
```
$ docker compose up
```

> This is the exemplary `docker-compose` file of this repository:

Reviewer: ```suggestion
This is the exemplary `docker-compose` file for the server:
```
